src/hpc/autoscale/job/job.py

```diff
@@ -35,6 +39,22 @@ def _check_type(**kwargs: Any) -> None:
     )
 
 
+def _initialize_job_config() -> None:
+    config_path = os.getenv("AUTOSCALE_JOB_CONFIG", "/opt/cycle/pbspro/allocate.conf")
```

Comment:
Why is this referencing pbs pro here?

Reply:
this is the location where logging.conf can be found; a relative path does not really work since the cwd of the hook is: /var/spool/pbs/server_priv/hooks/tmp; happy to pick a more generic location (/opt/cycle/scalelib?)

Reply:
it looks like the venv is being installed here by cyclecloud-pbspro; can be moved to /opt/cycle/scalelib, maybe...

Reply:
Maybe this is a question for @ryanhamel? If we have an /opt/cycle/scalelib directory already created (and it looks like we do), then I agree it should be in /opt/cycle/scalelib.